ci/conda/recipes/morpheus/meta.yaml

```diff
@@ -77,6 +78,8 @@ outputs:
         - dill
         - distributed
         - docker-py 5.0.*
+        - elasticsearch==8.9.0
+        - feedparser=6.0.10
```

Comment:
Same here, not every use will need these dependencies. Should we have an include guard instead of adding it to every install?

Reply:
Is your primary concern the dependencies which appear here in the `meta.yaml` as opposed to those in the `cuda11.8_dev.yml` file?